storage/interfaces.go

```diff
@@ -90,6 +90,8 @@ type NotificationSettings struct {
 	QueuesSize           int           `mapstructure:"queues_size"`
 	MinReconnectInterval time.Duration `mapstructure:"min_reconnect_interval"`
 	MaxReconnectInterval time.Duration `mapstructure:"max_reconnect_interval"`
+	CleanInterval        time.Duration `mapstructure:"clean_interval"`
```

Comment:
maybe these 2 need to be validated, because setting a negative `older_than will increase the timestamp in the clean functions and will clean the whole table?